nwb.config.js

```diff
@@ -1,19 +1,53 @@
-const browsers = require('./test/browsers');
+const babel = require('babel-core');
 const externals = require('webpack-node-externals');
-const [ cmd ] = require('yargs').argv._;
+const fs = require('fs');
+const glob = require('glob');
+const mkdirp = require('mkdirp');
+const path = require('path');
+const rimraf = require('rimraf');
+const yargs = require('yargs');
+
+const browsers = require('./test/browsers');
+
+const [ cmd ] = yargs.argv._;
+const isBuild = cmd === 'build';
+const babelConfig = {
+  plugins: ['transform-react-jsx']
+};
+
+if (isBuild) {
+  const files = glob.sync('src/**/*.js');
+  rimraf.sync('./es');
+  files.forEach(f => {
+    const { code } = babel.transformFileSync(f, Object.assign({}, babelConfig, {
+      presets: ['stage-0', 'es2017', 'es2016']
+    }));
+    const target = f.replace('src/', 'es/');
+    const targetDir = path.dirname(target);
+    mkdirp.sync(targetDir);
+    fs.writeFileSync(target, code);
+  });
+}
 
 module.exports = {
   type: 'web-module',
   npm: {
+    // CJS is kinda useless if we have UMD.
     cjs: false,
-    esModules: true,
+
+    // We create a custom ESBuild above.
```

Comment:
I'm confused, so the custom ESBuild will produce  es6 modules with transiled code to es5 ?
-> ` presets: ['stage-0', 'es2017', 'es2016']`

If so, this doesn't make any sense -> we don't want classes to be transformed to function